app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -395,4 +398,13 @@ def add_secret_to_pod_def(pod_def, inspector_admin_secret_name)
   def inspector_image
     'docker.io/openshift/image-inspector:2.0'
   end
+
+  def inspector_proxy_env_variables
+    settings = ext_management_system.custom_attributes
+    PROXY_ENV_VARIABLES.each_with_object([]) do |st, env|
```

Comment:
@Fryguy I don't think its working for me because `PROXY_ENV_VARIABLES` is a list of strings.